### PR TITLE
[FR] Business cards implementation

### LINKED ISSUE #14819: Add support for business cards

### PR DESCRIPTION
**Pull request by maririos**
Includes:
- implementation
- adds locale to prebuilt with no tests. Full coverage will be implemented as part of #14817
- tests
- samples
- feedback from service team on other samples

Fixes: #14819

### CODE REVIEW

File: sdk/formrecognizer/Azure.AI.FormRecognizer/CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@
 - Added ability to create a composed model from the `FormTrainingClient` by calling method `StartCreateComposedModel`.
 - Added the properties `ModelName` and `Properties` to types `CustomFormModel` and `CustomFormModelInfo`.
 - Added property `ModelName` to `TrainingOptions` and new type `CreateComposedModelOptions`.
+- Added support for pre-built business card recognition.
```
Comment (iscai-msft) [inline sdk/formrecognizer/Azure.AI.FormRecognizer/CHANGELOG.md:12]: nit: is there any way to make this one stand out more, it seems more important than adding properties. Maybe include it at the top of the list?

**Review by iscai-msft — APPROVED** [review]

**Review by kristapratico — APPROVED** [review]
Looks good. Although I do think we should add multipage business card tests once we have more good sample files

File: sdk/formrecognizer/Azure.AI.FormRecognizer/CHANGELOG.md (hunk 1 above)
Comment (maririos) [reply sdk/formrecognizer/Azure.AI.FormRecognizer/CHANGELOG.md]: Good point. 
Updated

**Pull request merged**